Makefile

```diff
@@ -126,9 +127,6 @@ manager: ## Build manager binary.
 $(KUSTOMIZE): $(TOOLS_DIR)/go.mod # Build kustomize from tools folder.
 	cd $(TOOLS_DIR); go build -tags=tools -o $(BIN_DIR)/kustomize sigs.k8s.io/kustomize/kustomize/v3
 
-$(CLUSTERCTL): go.mod ## Build clusterctl binary.
-	go build -o $(BIN_DIR)/clusterctl sigs.k8s.io/cluster-api/cmd/clusterctl
-
```

Comment:
@jichenjc I cant understand why do we need to remove this.
> 2020-09-29 10:17:00.179441 | ubuntu-bionic-large | wget: bin/clusterctl: No such file or directory
2020-09-29 10:17:00.180659 | ubuntu-bionic-large | Makefile:313: recipe for target 'create-cluster' failed
2020-09-29 10:17:00.180693 | ubuntu-bionic-large | make: *** [create-cluster] Error 1

The build might be failing due to this.